crud.py

```diff
@@ -0,0 +1,36 @@
+from sqlalchemy.orm import Session
+
+import models
+import schemas
+
+
+def create_author(db: Session, author: schemas.AuthorCreate):
+    db_author = models.DBAuthor(name=author.name, bio=author.bio)
```

Comment:
Ensure that `models.DBAuthor` is defined in the `models` module and has the attributes `name` and `bio`.